examples/sugarscape.py

```diff
@@ -0,0 +1,261 @@
+# Sugarscape
+#
+# To run, you must install pygame:
+# pip install pygame
+#
+# Original paper:
+# Epstein, J.M. & Axtell, R. Artificial Life and Robotics (1997) 1: 33. https://doi.org/10.1007/BF02471109
+#
+# This reproduces the model according to NetLogo.
+#
+# The pygame visualization seems to have a busy wait loop which will peg a processor.
```

Comment:
I looked into that. If I remember correctly pygame did have a busy loop.

Reply:
Ahh okay, that comment was copied from one of the other examples, should I change it?